recipe/recipe.yaml

```diff
@@ -0,0 +1,576 @@
+          then:
+            - llvm-openmp
+        - if: linux
+          then:
+            - libgomp
+        - libblas
+        - libcblas
+        - liblapack
+      ignore_run_exports:
+        from_package:
+          - libblas
+          - libcblas
+          - liblapack
+          - ${{ fortran_compiler }}
+      run_exports:
+        - ${{ pin_subpackage('libsuitesparseconfig') }}
+    tests:
+      - package_contents:
+          lib:
+            - suitesparseconfig
+  - package:
+      name: libamd
+      version: 3.3.3
+    about:
+      license: BSD-3-Clause
+      license_file: AMD/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
+        env:
+          SUBPKG_DIR: AMD
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - ${{ compiler('fortran') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - ${{ pin_subpackage('libsuitesparseconfig', exact=True) }}
+      run_exports:
+        - ${{ pin_subpackage('libamd') }}
+    tests:
+      - package_contents:
+          lib:
+            - amd
+  - package:
+      name: libbtf
+      version: 2.3.2
+    about:
+      license: LGPL-2.1-or-later
+      license_file: BTF/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
+        env:
+          SUBPKG_DIR: BTF
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - ${{ pin_subpackage('libsuitesparseconfig', exact=True) }}
+      run_exports:
+        - ${{ pin_subpackage('libbtf') }}
+    tests:
+      - package_contents:
+          lib:
+            - btf
+  - package:
+      name: libcamd
+      version: 3.3.3
+    about:
+      license: BSD-3-Clause
+      license_file: CAMD/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
+        env:
+          SUBPKG_DIR: CAMD
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
+        - pkg-config
+      host:
+        - ${{ pin_subpackage('libsuitesparseconfig', exact=True) }}
+      run_exports:
+        - ${{ pin_subpackage('libcamd') }}
+    tests:
+      - package_contents:
+          lib:
+            - camd
+  - package:
+      name: libccolamd
+      version: 3.3.4
+    about:
+      license: BSD-3-Clause
+      license_file: CCOLAMD/Doc/License.txt
+    build:
+      script:
+        file: ${{ 'build-one.sh' if not win else 'build-one.bat' }}
+        env:
+          SUBPKG_DIR: CCOLAMD
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
```

Comment:
nice 👀

Reply:
I am sure you know, but inline jinja should also work (`${{ "ninja" if win else "make" }}`)

Reply:
For rattler-build it doesn't really matter as we have to parse the `jinja` AST in both cases anyways (the `if: <cond>` is also evaluated with minijinja, and we extract any used variables from there before rendering the recipe).

Reply:
Yeah, I had the model in mind that when conda-smithy might be collecting the list of dependencies, it can concatenate all the lists and ignore the conditionals, assuming every branch will be taken by _some_ build. But that may not have anything to do with what actually happens. And presumably it still needs to evaluate jinja for every build matrix entry anyway for completeness, so it probably doesn't make a difference.

I just get the _feeling_ that it's nice to avoid templates where possible in the new format, even if it doesn't actually matter in practice.